### PR TITLE
Pal 543 create k8s job to run example

### PR DESCRIPTION
**Pull request by developer01189998819991197253**

### CODE REVIEW

**Review by m78233 — DISMISSED** [review]

File: data-service/src/main/java/uk/gov/gchq/palisade/service/data/config/ApplicationConfiguration.java [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2019 Crown Copyright
+ * Copyright 2020 Crown Copyright
```
Comment (ac74475) [inline data-service/src/main/java/uk/gov/gchq/palisade/service/data/config/ApplicationConfiguration.java]: revert, there are no other changes to this file

File: licenses/eclipse_public_license_1.0.html [hunk 2]
```diff
@@ -15,20 +15,20 @@
     <link rel="shortcut icon" href="/eclipse.org-common/themes/solstice/public/images/favicon.ico"/>
```
Comment (ac74475) [inline licenses/eclipse_public_license_1.0.html]: remove this licence change

File: resource-service/src/main/java/uk/gov/gchq/palisade/service/resource/service/StreamingResourceServiceProxy.java [hunk 3]
```diff
@@ -171,11 +171,11 @@ public Boolean addResource(final LeafResource leafResource) {
     public void initPostConstruct() {
         // Add resources to persistence
         LOGGER.info("Prepopulating using resource builder: {}", resourceBuilder);
-        resourceBuilder.get().stream()
-                .peek(entry -> LOGGER.debug(entry.toString()))
+        resourceBuilder.get()
                 .forEach(entry -> {
                     Resource rootResource = entry.getKey();
                     LeafResource leafResource = entry.getValue();
+                    LOGGER.info("Persistence add for {} -> {}", rootResource.getId(), leafResource.getId());
```
Comment (ac74475) [inline resource-service/src/main/java/uk/gov/gchq/palisade/service/resource/service/StreamingResourceServiceProxy.java]: only really want this to be logged at debug level in a production system

File: values.yaml [hunk 4]
```diff
@@ -65,6 +65,15 @@ global:
...
   deployment: codeRelease
 
+  # set environment variables for all palisade services pods by deployment type eg. spring profiles active
+  env:
+    _default:
+    - name: "SPRING_PROFILES_ACTIVE"
+      value: "k8s"
+    codeRelease:
+    - name: "SPRING_PROFILES_ACTIVE"
+      value: "k8s"
```
Comment (ac74475) [inline values.yaml]: why a _default and a code release?

**Review by ac74475 — CHANGES_REQUESTED** [review]

File: resource-service/src/main/java/uk/gov/gchq/palisade/service/resource/service/StreamingResourceServiceProxy.java (hunk 3 above)
Comment (dev930018) [reply resource-service/src/main/java/uk/gov/gchq/palisade/service/resource/service/StreamingResourceServiceProxy.java]: In all other services this is the current behaviour, changed for consistency.
Could have all changed to DEBUG but as we are changing some caching things, i think it's useful to keep at info for a bit

File: values.yaml (hunk 4 above)
Comment (dev930018) [reply values.yaml]: Set different profiles depending upon deploy type. By default we want to run the jars with the k8s profile, on codeRelease we also want the k8s profile, but in example deployment we want to have example-prepopulation data loaded, so need the example-libs profile AND k8s

File: data-service/src/main/java/uk/gov/gchq/palisade/service/data/config/ApplicationConfiguration.java (hunk 1 above)
Comment (developer01189998819991197253) [reply data-service/src/main/java/uk/gov/gchq/palisade/service/data/config/ApplicationConfiguration.java]: removed

File: licenses/eclipse_public_license_1.0.html (hunk 2 above)
Comment (developer01189998819991197253) [reply licenses/eclipse_public_license_1.0.html]: removed

File: values.yaml (hunk 4 above)
Comment (ac74475) [reply values.yaml]: delete code release one then

**Review by ac74475 — CHANGES_REQUESTED** [review]
Just need to delete the 3 lines in the values.yaml for the codeRelease env variables so we just use the defaults. Given they are both the same

File: values.yaml (hunk 4 above)
Comment (developer01189998819991197253) [reply values.yaml]: deleted

**Review by ac74475 — APPROVED** [review]

**Pull request merged**